### ISSUE TITLE
Update README description of abandoned

### ISSUE THREAD

**Issue by JohnTitor** [notify-rs/notify#221]
This crate is looking for maintainers and we have some maintainers now.
It'd be nice to have the description for looking for maintainers instead of the abandoned description on README.

**Comment by dhiemaz** [comment]
Nice catch.. @JohnTitor we can update readme to remove 'Abandoned' description.

**Comment by Geobert** [comment]
Just to let you guys know that the text is still showing on crates.io (I was confused by the state of the crate and found this github issue :))

**Comment by Dylan-DPC** [comment]
@Geobert sorry about it. But yeah it will be reflected only after we do a release which we haven't done yet since we created the new organisation.
